hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/builder/StatusIconBuilder.java

```diff
@@ -86,7 +87,7 @@ public ActionStatusIconSupplier(final VaadinMessageSource i18n, final Function<E
             addMapping(Status.CANCEL_REJECTED, VaadinIcons.EXCLAMATION_CIRCLE, SPUIStyleDefinitions.STATUS_ICON_ORANGE);
             addMapping(Status.CANCELED, VaadinIcons.CLOSE_CIRCLE, SPUIStyleDefinitions.STATUS_ICON_GREEN);
             addMapping(Status.ERROR, VaadinIcons.EXCLAMATION_CIRCLE, SPUIStyleDefinitions.STATUS_ICON_RED);
-            addMapping(Status.WAIT_FOR_CONFIRMATION, VaadinIcons.USER, SPUIStyleDefinitions.STATUS_ICON_PENDING);
+            addMapping(Status.WAIT_FOR_CONFIRMATION, VaadinIcons.USER_CLOCK, SPUIStyleDefinitions.STATUS_ICON_PENDING);
```

Comment:
I think we dont use the same icon when we do manual and rollouts assignment to reflect in action status right? Wouldn't it be better to unify it across all the assignment types?

Reply:
Will be discussed in the feature demo.